doc/schemas/pm-context-schema.json

```diff
@@ -21,13 +22,16 @@
         "contextGroupName": {
           "type": "string"
         },
+        "comment": {
+          "type": "string"
+        },
         "UMLConceptUID": {
           "type": "string"
         },
         "SNOMEDConceptID": {
           "type": "string"
         },
-        "MeasurementUnitsCodes": {
+        "MeasurementUnitsCode": {
```

Comment:
I think you already told me about this, but I forgot. Glad to know it is not the issue of validator!